CONTRIBUTING.md

```
@@ -0,0 +1,210 @@
+
+Write clear, descriptive commit messages:
+
+```bash
+git add .
+git commit -m "Add feature: describe what you added"
+```
+
+**Commit message guidelines:**
+This project follows Conventional Commits.
+All commit messages must conform to the Conventional Commits specification:
+https://www.conventionalcommits.org/en/v1.0.0/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)